packages/page-democracy/src/Overview/PreImage.tsx

```diff
@@ -80,8 +80,8 @@ function PreImage ({ className = '', imageHash, isImminent: propsIsImminent, onC
               onChange={setProposal}
             />
             <Input
+              className={'disabledLook'}
```

Comment:
```suggestion
              className='disabledLook'
```

Generally would prefer an actual flag instead of className pushed in - that way it can be checked. However can see how this holds up and then move to base at a later point in time. (Inputs will be replaced as they are implemented, so not worth scratching around in there too much atm)